pkg/graveler/graveler.go

```diff
@@ -803,6 +807,14 @@ func (g *Graveler) DeleteBranch(ctx context.Context, repositoryID RepositoryID,
 	return err
 }
 
+func (g *Graveler) GetStagingToken(ctx context.Context, repositoryID RepositoryID, branchID BranchID) (*StagingToken, error) {
+	branch, err := g.RefManager.GetBranch(ctx, repositoryID, branchID)
+	if err != nil {
+		return nil, err
+	}
+	return &branch.StagingToken, nil
```

Comment:
so there is no nil option or do we need to check for empty string and return nil

Reply:
For now we always have a staging token.